GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3563,6 +3564,33 @@ private void FillTerminalTab()
 		    };
 	    }
 
+        public void ChangeTerminalActiveFolder(string path)
+        {
+            if (terminal == null || terminal.RunningSession == null || string.IsNullOrWhiteSpace(path))
+                return;
+
+            string posixPath;
+            if (TryConvertWindowsPathToPosix(path, out posixPath))
+            {
+                //Clear terminal line
```

Comment:
In fact, once the conemu session is created, there is no other way than simulate keyboard input to let it run a command.

This code simulate the press on backspace 10000 times to remove all the already typed text if there is one (otherwise it will make fail the command to run). I have tested, it works and there is no performance problem.

The other solution would have been to destroy the session and recreate one in the good folder but I think that my solution is a lot better because it permit to keep the terminal history if the user want to use the same command in different repositories...(which I sometimes do.)